data/human/human missions.txt

```diff
@@ -7453,3 +7454,794 @@ mission "Adelita 5"
+			`	"So what can I help you with..." Deoric Drifa's voice trails off as he sees Ildico.`
+			`	Ildico tries to step forward than stops herself. She freezes up.`
+			`	"It can't be," the elder Drifa whispers.`
+			`	"Why. Didn't. You. Come?" Ildico forces out each word.`
+			`	"I sent for you. They told me you were all dead. They found bodies... and the generator blown."`
+			`	"The accident..." Ildico says more to herself than anyone else. And then in that instant she unfreezes, emotions roil through her, and she runs to her father. Everything forgiven.`
+			`	After a long tear-filled embrace she disengages and wipes her eyes. "Early on the generator blew and we had to evacuate the compound. Some people died.. including Hogne." She glances at you and says, "He was my older brother."`
+			`	She continues, "I salvaged what I could and set up at home. I've been at home the whole time."`
+			`	"It's my fault," Deoric says. "I should have gone and looked for you myself. I would have known where to look."`
+			`	They talk about family, the time apart, and what happened back on Hope. Finally, Ildico turns her attention back to you.`
+			`	"Thank you for bringing me here, <first>. For sticking with me. I appreciate it." She gives you the first real genuine smile you've seen on her face.`
+			choice
+				`	"No problem, Ildico."`
+				`	"You're welcome, Ildico."`
+
+			`	She looks at you for just a moment, as if in thought, and then says, "If we ever meet again, call me Ildy. It's what my friends call me."`
+			`	With that you take your leave.`
+				decline
+
+
+event "deoricdeeds"
+
+
+mission "Ice Queen 5"
+	landing
+	name "Visit Ildico"
+	description "Ildico has asked you to visit her on <destination>."
+	source
+		government "Republic" "Free Worlds" "Syndicate" "Neutral" "Independent"
+	destination "Chiron"
+	to offer
+		has "event: deoricdeeds"
+
+	on offer
+		dialog
+			`Your inbox blinks with a new message from Ildico on Chiron:`
+			``
+			`	"I need to see you. Please come. My address is attached.`
+			`				-your friend`
+			`					Ildy"`
+
+	on complete
+		dialog
+			`The myriad lights of urban settlements twinkle in the dusk as you approach the capital, and main spaceport, of Chiron. Crickets, brought from old Earth, chirp in the twilight air. Your ship parked, you'll need to make your way to the main spaceport to hire transportation to Ildico's address.`
+
+
+mission "Ice Queen 6"
+	name "Take Ildico to <planet>"
+	description "Smuggle Ildico and her child to <destination>."
+	source "Chiron"
+	destination "Haven"
+	passengers 2
+	illegal 1250000 `Ildico Drifa is dragged away in chains for high crimes against the Republic and its most honorable servants. Her infant child is taken into the custody of the Republic. You can only watch in sad silence as they are removed from your ship. You wind up quite a bit poorer through a combination of fines and bribes (the breakdown remains somewhat unclear).`
```

Comment:
This could run into problems where the Wanderers or Heliarch fine you for high treason against the Republic.
Probably not something that can be fixed at the moment, but just thought I'd point it out.

Reply:
:|